poc_iot_verifier/src/runner.rs

```diff
@@ -147,8 +147,12 @@ impl Runner {
         let beacon_len = db_beacon_reports.len();
         tracing::info!("found {beacon_len} beacons ready for verification");
         for db_beacon in db_beacon_reports {
+            let entropy_start_time = match db_beacon.timestamp {
+                Some(v) => v,
+                None => return Ok(())
```

Comment:
this is going to stop processing _all_ the beacons in the set returned; do you want to do that or just skip over the invalid beacon and continue pulling associated witnesses for the valid ones?

Reply:
bah, yeah it should just skip the current one.  Fix incoming

Reply:
fixed here: f0a987d14808969b49debf76c98da275f91d40c8